code/modules/atmospherics/FEA_fire.dm

```diff
@@ -91,196 +90,200 @@
 	var/datum/light/light
 #endif
 
-	animate_movement = NO_STEPS // fix for weird unpool sliding
-
 	var/volume = 125
 	var/temperature = FIRE_MINIMUM_TEMPERATURE_TO_EXIST
-	var/just_spawned = 1
-	var/bypassing = 0
+	/// If we've just spawned, don't die right after, wait a cycle.
+	var/just_spawned = FALSE
```

Comment:
You seem to have changed just_spawned from 1 to `FALSE`. Will that break things (is it supposed to be `TRUE`) or is that intended?

Reply:
ah its not intended, fixing